### ISSUE TITLE
Adapter seems to ignore a 'filename' parameter

### ISSUE THREAD

**Issue by namelessichi** [waterlinejs/sqlite3-adapter#8]
Hi, 

I don't know if it's a bug actually because don't know the original idea. I did't get if this project is still supported also.

But anyway it would be nice to have here:

```
sqlite3-adapter/lib/adapter.js:85
```

something like

``` js
filename: !_.isEmpty(filename) && filename != ':memory:' ? filename : ('.tmp/' + connection.identity + '.sqlite')
```

instead of

``` js
filename: '.tmp/' + connection.identity + '.sqlite'
```

**Comment by KSDaemon** [comment]
Yep! Confirming, that db filename parameter from connections.js (i'm using sails.js) is ignored. Spend a few hours investigating that... :(

**Comment by eyasliu** [comment]
so, I have to use version 0.12.4，the filename is work

**Comment by mbret** [comment]
Yes, it's a real problem :/. Any news about futur update @tjwebb ?

note: I'm using waterline outside of sails.

**Comment by matebordas** [comment]
+1 Same issue.

**Comment by morungos** [comment]
+1 from here too. A fix would be nice

**Comment by zuhrasofyan** [comment]
is this issue abandoned? I've used this and since the file stored in tmp file, all my saved data work vanished once I start the server :/

**Comment by tjwebb** [comment]
> I've used this and since the file stored in tmp file, all my saved data work vanished once I start the server :/

That is not related to this issue. just being in `.tmp` doesn't mean your data gets deleted. You need to set your `migrate` config properly in Sails.
